recipes/smallgenomeutilities/0.2/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "smallgenomeutilities" %}
+{% set version = "0.2" %}
+{% set md5 = "f8b222892e9f18bb135efd5802a878a4" %}
+{% set sha256 = "802ca5f85a14e8243aa44e5f4fb771691f0c80689cf3ecb1edce5382ce0ecda3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/f8/e2/54b075294a3247595abf9f7d7314eddc04c77a049398157514b65087fe6a/{{ name|lower }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # not 'noarch python', because of skip version, and because of command line utils
+  skip: True # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - biopython
+    - numpy
+    - progress
+    - pysam
+    - scikit-learn # sklearn
+    - scipy
+    - matplotlib
+
+  run:
+    - python
+    - biopython
+    - numpy
+    - progress
+    - pysam
+    - scikit-learn # sklearn
+    - scipy
+    - matplotlib
+
+test:
+  imports:
+    - smallgenomeutilities
+
+  commands:
+    - convert_qr -h
+    - convert_reference -h
+    - coverage -h
+    - coverage_stats -h
+    - extract_coverage_intervals -h
+    - extract_sam -h
+    - extract_seq -h
+    - mapper -h
+    - minority_freq -h
+    - pair_sequences -h
+    - predict_num_reads -h
+    - remove_gaps_msa -h
+    - MPLBACKEND="Agg" compute_mds  -h
+      # NOTE the default matplotlib backend used in Linux (Qt) is openGL-accelerated. Use the pure software "Agg" instead for testing inside GL-less containers
+
+about:
+  home: https://github.com/cbg-ethz/smallgenomeutilities
+  license: GNU General Public License v2 or later (GPLv2+)
```

Comment:
For GPL, it woulde be nice to include a `license_file:`.
I filled an issue upstream.

Reply:
Okay, I'll see with @sposadac to add one in the repo.

Reply:
@DrYak Please change this to add 0.2.1, which has proper metadata added and includes the `LICENSE` file.

Reply:
Added upstream @SoapZA.